code/modules/mob/mob_helpers.dm

```diff
@@ -258,62 +258,58 @@
  * * notify_suiciders If it should notify suiciders (who do not qualify for many ghost roles)
  * * notify_volume How loud the sound should be to spook the user
  */
-/proc/notify_ghosts(message, ghost_sound, enter_link, atom/source, mutable_appearance/alert_overlay, action = NOTIFY_JUMP, flashwindow = TRUE, ignore_mapload = TRUE, ignore_key, header, notify_suiciders = TRUE, notify_volume = 100) //Easy notification of ghosts.
+/proc/notify_ghosts(
+	message, \
+	ghost_sound, \
+	enter_link, \
+	atom/source, \
+	mutable_appearance/alert_overlay, \
+	action = NOTIFY_JUMP, \
+	flashwindow = TRUE, \
+	ignore_mapload = TRUE, \
+	ignore_key, \
+	header, \
+	notify_suiciders = TRUE, \
+	notify_volume = 100 \
+) //Easy notification of ghosts.
```

Comment:
```suggestion
/proc/notify_ghosts(
	message,
	ghost_sound,
	enter_link,
	atom/source,
	mutable_appearance/alert_overlay,
	action = NOTIFY_JUMP,
	flashwindow = TRUE,
	ignore_mapload = TRUE,
	ignore_key,
	header,
	notify_suiciders = TRUE,
	notify_volume = 100
)
```

no need for backslashes when it's a proc, we only really need backslashes in macro-wrapped stuff like `AddComponent` and `AddElement`. BYOND-standard stuff works natively with multilining. Also that comment is not useful so let's axe it.